src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Common.Test/Microsoft.AspNetCore.Razor.LanguageServer.Common.Test.csproj

```diff
@@ -6,6 +6,7 @@
 
   <ItemGroup>
     <None Include="xunit.runner.json" CopyToOutputDirectory="PreserveNewest" />
+    <Compile Include="..\OSSkipConditionFactAttribute.cs" LinkBase="Shared" />
```

Comment:
Why not compile the new `class` into Microsoft.AspNetCore.Razor.Test.Common❔

Reply:
Not all test projects reference from what I remember.